phpunit.xml.dist

```diff
@@ -3,10 +3,9 @@
          colors="true"
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         syntaxCheck="true">
+         convertWarningsToExceptions="true">
     <testsuite name="transcoder-bundle">
-        <directory>tests</directory>
+        <directory suffix=".php">tests</directory>
     </testsuite>
     <filter>
```

Comment:
@timonf filter bitte rausschmeißen. ggf. phpunit version in phpunit.xml eintragen

Reply:
@jojoguru Done.